scim/data_account_group.go

```diff
@@ -0,0 +1,93 @@
+package scim
+
+import (
+	"context"
+	"sort"
+	"strings"
+
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+// DataSourceGroup returns information about group specified by display name
+func DataSourceAccountGroup() *schema.Resource {
+	type entity struct {
```

Comment:
What about adding `id` as well? It would be really useful, especially for groups assignment to workspaces